### PR TITLE
Add a region migration tutorial.

### PR DESCRIPTION
**Pull request by Crell**
It's all manual, but at least it's explained and we can point people at it now.

### CODE REVIEW

File: src/tutorials/region-migration.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Region migration
+
+Platform.sh is available in a number of different *Regions*.  Each region is a self-contained copy of Platform.sh in a single datacenter.  When you first create a project you can specify what region it should be in.
```
Comment (mrkschan) [inline src/tutorials/region-migration.md]: em... i'm not sure .... `what region` v.s. `which region`?

File: src/tutorials/region-migration.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+There is no automated way to migrate a project from one region to another after it is created.  However, the process to do so manually is fairly straightforward.
+
+## Why migrate regions?
```
Comment (mrkschan) [inline src/tutorials/region-migration.md]: shall this be "migrate between regions"?

File: src/tutorials/region-migration.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* Different datacenters are located in different geographic areas, and you may want to keep your site physically close to the bulk of your user base for reduced latency.
+* Only selected regions offer European Sovereign hosting.  If you created a project in a non-Sovereign region you may need to migrate to a Sovereign region.
+* Some regions are running older versions of the Platform.sh codebase that offers fewer features.  In particular, the US-1 and EU-1 regions do not offer XL and 2XL plans, self-terminating builds in case of a build process that runs too long, or distributing environments across different grid hosts.  If you are on one of those regions and desire those features you will need to migrate to the newer US-2 or EU-2 regions.
```
Comment (mrkschan) [inline src/tutorials/region-migration.md]: shall we replace "codebase" with "orchestration system"?

File: src/tutorials/region-migration.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Make a Git clone of your existing project.  Then add a Git remote to the new project, using the Git URL shown in the UI.  Push the code for at least your master branch to the new project.  (You can also transfer other branches if desired.  That's optional.)
+
+Copy your existing user files on the old project to your computer using `rsync`.  See the [exporting](/tutorials/exporting.md) page for details.  Then use `rsync` to copy them to the same directory on the new project.  See the [migrating](/tutorials/migrating.md) page for details.
+
+Export your database from the old project and import it into the new project.  Again, see the exporting and migration pages, as well as the instructions for your specific database services.
+
+Re-enter any project or environment variables you've defined on your old project in your new project.
+
+Add any users to your new project that you want to continue to have access.
```
Comment (mrkschan) [inline src/tutorials/region-migration.md]: Shall we remind customer to create Integrations on new project as well, especially the disk usage notification?

File: src/tutorials/region-migration.md (hunk 1 above)
Comment (Crell) [reply src/tutorials/region-migration.md]: I think which is probably more correct, although both are colloquially common.  I'll fix.

File: src/tutorials/region-migration.md (hunk 3 above)
Comment (Crell) [reply src/tutorials/region-migration.md]: I like that better, yes.

File: src/tutorials/region-migration.md (hunk 4 above)
Comment (Crell) [reply src/tutorials/region-migration.md]: Good call!  I'll add.

**Pull request merged**